### DISCUSSION TITLE
JOSS manuscript comments

### DISCUSSION THREAD

**Discussion by kellypierce** [UT-Covid/episimlab#54] [General]
https://github.com/UT-Covid/episimlab/blob/ce4b8802b3f294dc949fa9e42b8b98d976b8d3fc/paper.md?plain=1#L26
Change "are therefore well suited to" to "can be applied to" (since the examples listed require additional infrastructure on top of the compartmental model)

https://github.com/UT-Covid/episimlab/blob/ce4b8802b3f294dc949fa9e42b8b98d976b8d3fc/paper.md?plain=1#L29
Consider changing "viral" to "pathogen" genomics.

https://github.com/UT-Covid/episimlab/blob/ce4b8802b3f294dc949fa9e42b8b98d976b8d3fc/paper.md?plain=1#L30
Don't end sentence with conjunction.

https://github.com/UT-Covid/episimlab/blob/ce4b8802b3f294dc949fa9e42b8b98d976b8d3fc/paper.md?plain=1#L39
Are there any constraints on the topology of the graph? It can be cyclic, right (e.g. an SIRS model)

https://github.com/UT-Covid/episimlab/blob/ce4b8802b3f294dc949fa9e42b8b98d976b8d3fc/paper.md?plain=1#L40
It might be helpful to provide a brief example here, along the lines of "... varying dimensions such as age-depended recovery rates, location-dependent contact patterns, and other demographically stratified risk factors." (but shorten if that seems too long)

https://github.com/UT-Covid/episimlab/blob/ce4b8802b3f294dc949fa9e42b8b98d976b8d3fc/paper.md?plain=1#L41
Change to "Simulate interventions, including dynamic interventions such as ...."

https://github.com/UT-Covid/episimlab/blob/ce4b8802b3f294dc949fa9e42b8b98d976b8d3fc/paper.md?plain=1#L44
"approachable" and "easily" are somewhat subjective; maybe replace approachable with "extensible" and omit easily

https://github.com/UT-Covid/episimlab/blob/ce4b8802b3f294dc949fa9e42b8b98d976b8d3fc/paper.md?plain=1#L45
You could reframe as "different or more complex"

https://github.com/UT-Covid/episimlab/blob/ce4b8802b3f294dc949fa9e42b8b98d976b8d3fc/paper.md?plain=1#L59
What do you mean by "Markov state matrix" here? 

https://github.com/UT-Covid/episimlab/blob/ce4b8802b3f294dc949fa9e42b8b98d976b8d3fc/paper.md?plain=1#L63
Not sure about this citation (I'll find one from the epi literature to replace). Recommend replacing "Markovian" with "deterministic or stochastic". Also specify that transitions are drawn from a Poisson distribution. (I think the term "Markovian" is correct here, but I don't typically hear these types of models described as such. And I suppose it's technically possible to have a non-Markovian stochastic implementation...)

https://github.com/UT-Covid/episimlab/blob/ce4b8802b3f294dc949fa9e42b8b98d976b8d3fc/paper.md?plain=1#L70
Replace "forecasting" with "projecting"

https://github.com/UT-Covid/episimlab/blob/ce4b8802b3f294dc949fa9e42b8b98d976b8d3fc/paper.md?plain=1#L103
Need to add citations for the [partition_v1](https://github.com/UT-Covid/episimlab/blob/joss/episimlab/models/partition_v1.py) model and for the [vaccine](https://github.com/UT-Covid/episimlab/blob/joss/episimlab/models/vaccine.py) model. (I'll find and add)

**Comment by ethho** [comment]
Pushed suggested revisions to #52 (9df32d7)

## Remaining To-Do

- [x] https://github.com/UT-Covid/episimlab/blob/ce4b8802b3f294dc949fa9e42b8b98d976b8d3fc/paper.md?plain=1#L63
    - [x] Epi citation for Gillespie algorithm/Markovian models
- [ ] https://github.com/UT-Covid/episimlab/blob/ce4b8802b3f294dc949fa9e42b8b98d976b8d3fc/paper.md?plain=1#L103
    - [ ] Mention pre-packaged models `partition_V1` and `vaccine`
    - [ ] Add citations for pre-packaged models `partition_V1` and `vaccine` to `paper.bib`
- [x] Alyssa in acknowledgements
- [x] Gleamviz section
- [x] Schedule 1 on 1 with Lauren to discuss authorship

**Comment by ancantu** [comment]
As a reader not entirely privy to epidemiological modeling:

https://github.com/UT-Covid/episimlab/blob/ce4b8802b3f294dc949fa9e42b8b98d976b8d3fc/paper.md?plain=1#L87
SIR (mentioned in other lines as well), SIS, and SEIR are mentioned without any explanation for what the acronyms stand for. For an audience that is not familiar with these models, it might be helpful to write out what they stand for.

https://github.com/UT-Covid/episimlab/blob/ce4b8802b3f294dc949fa9e42b8b98d976b8d3fc/paper.md?plain=1#L58
The explanation prior to this line about how other packages have limitations reasonably demonstrates why there is a need for Episimlab. However, I feel that this line (line 58) does not accurately reflect that your SDK will make up for the limitations present in other packages. I would reword to make this more clear to a general audience.

Overall, I could reasonably follow along for a substantial amount of it, and I believe addressing the comments above will help for a general audience.

**Comment by ethho** [comment]
@kellypierce I'm looking around on the [consortium publications page](https://covid-19.tacc.utexas.edu/publications/) for the citations for `partiton_v1` and `vaccine` models... Are these two from Suzanna the ones we'd want to cite?

- https://doi.org/10.15781/cytm-ev55
- https://doi.org/10.15781/503k-c813

**Reply by kellypierce** [reply]
@eho-tacc here are some citations: 

This manuscript is the first peer-reviewed article that describes the model implemented in `partition_v`: https://www.nature.com/articles/s41467-021-23989-x

This report is the first report that describes the vaccination model implemented in `vaccine` (nothing peer-reviewed yet):
https://doi.org/10.15781/gc9z-9h56

There may be other reports and papers we can cite as well; we can talk to other folks in the lab about how they'd prefer these works to be cited.
